packages/sdk/src/across/clients/bridgePool.ts

```diff
@@ -522,7 +527,17 @@ export class Client {
     const latestBlock = await this.deps.provider.getBlock("latest");
     const previousBlock = await this.deps.provider.getBlock(latestBlock.number - blockDelta);
     const state = await pool.read(latestBlock.number, previousBlock.number);
-    this.state.pools[poolAddress] = joinPoolState(state, latestBlock, previousBlock);
+
+    let rateModel: RateModel | undefined = undefined;
+    try {
+      const rateModelRaw = await this.rateModelInstance.callStatic.l1TokenRateModels(state.l1Token);
+      rateModel = parseAndReturnRateModelFromString(rateModelRaw);
+    } catch (err) {
+      // caller can optionally listen error but we can continue
```

Comment:
>  we could swallow this error or just log it since getting the rate model is optional,
>  but we will just emit it to the caller and let them decide what to do with it.